src/snowflake/cli/plugins/nativeapp/run_processor.py

```diff
@@ -258,10 +258,10 @@ def drop_application_before_upgrade(self, policy: PolicyBase, is_interactive: bo
         )
         if not policy.should_proceed(user_prompt):
             if is_interactive:
-                print("Not upgrading the application.")
+                cc.step("Not upgrading the application.")
```

Comment:
not sure that's a step...

Reply:
I mentioned in the description the guide I am using to decide between `step` and `warning`, adding that here too:
```
How I have used each of these:

1. If there any internal checks done on the objects (such as checking its distribution property, or owner role) and a mismatch is found, then warning is used.
2. For regular progress, step is used.
3. If a user interacts with the CLI, then the expected outcome is also captured as a step.
phase is used only once where it made sense to me.
```
I believe that when a user accepts or rejects a prompt from the CLI, then what is output is just information to them because it is an expected outcome. It is not a warning that the user needs to be alarmed about. 

In general, I think we need something like `cc.info` because I do agree with the argument that not all information is a step.